src/App.js

```diff
@@ -1,31 +1,36 @@
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import { SignedInRoute } from './Components/middlewares';
 import {
-  Signin, Register, PasswordReset, NewPassword,
+  Signin,
+  Register,
+  PasswordReset,
+  NewPassword,
 } from './Components/authentication';
```

Comment:
These components are inside `src/components/` not `src/Components/`. Kindly fix that as it's causing conflicts with your linters.